frontend/src/employee-frontend/components/reports/TitaniaErrors.tsx

```diff
@@ -0,0 +1,101 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+// import { Link } from 'react-router-dom'
+// import styled from 'styled-components'
+
+import {
+  TitaniaErrorReportRow,
+  TitaniaErrorUnit,
+  TitaniaErrorEmployee
+} from 'lib-common/generated/api-types/reports'
+// import HelsinkiDateTime from 'lib-common/helsinki-date-time'
+// import LocalDate from 'lib-common/local-date'
+import { useQueryResult } from 'lib-common/query'
+import Title from 'lib-components/atoms/Title'
+// import { MutateButton } from 'lib-components/atoms/buttons/MutateButton'
+import ReturnButton from 'lib-components/atoms/buttons/ReturnButton'
+import { Container, ContentArea } from 'lib-components/layout/Container'
+import { Table, Thead, Th, Tbody, Td, Tr } from 'lib-components/layout/Table'
+// import { FixedSpaceRow } from 'lib-components/layout/flex-helpers'
+// import ExpandingInfo from 'lib-components/molecules/ExpandingInfo'
+import { H1, H2, H3 } from 'lib-components/typography'
+import { Gap } from 'lib-components/white-space'
+
+import { useTranslation } from '../../state/i18n'
+import { renderResult } from '../async-rendering'
+
+// import { TableScrollable } from './common'
+import { titaniaErrorsReportQuery } from './queries'
```

Comment:
Remove unused imports, don't just comment them out

Reply:
Oops, forgot that.